pymc/distributions/multivariate.py

```diff
@@ -1413,14 +1413,32 @@ class LKJCholeskyCov:
     """
 
     def __new__(cls, name, eta, n, sd_dist, *, compute_corr=True, store_in_trace=True, **kwargs):
+        dims = kwargs.pop("dims", None)
+
+        if dims is not None:
```

Comment:
Not sure this should be done by default. If users want to resize the model they will have to know there is a special dims they also need to update